Casks/p/pdfpals.rb

```diff
@@ -0,0 +1,28 @@
+cask "pdfpals" do
+  version "1.6.0"
+  sha256 "e082ff0565b8503ff6ab76c7d9d796f87a5631590732e3a2fec2b5b25e46720c"
+
+  url "https://download.pdfpals.com/releases/PDFPals-#{version}.dmg"
+  name "PDF Pals"
+  desc "AI Chat with PDFs"
```

Comment:
```suggestion
  desc "PDF Viewer"
```
Per our [guidelines](https://docs.brew.sh/Cask-Cookbook#stanza-desc).

Reply:
Another option is "AI PDF Analysis" but it doesn't sound very natural. What do you think @razvanazamfirei